static/js/apps/nl_interface/app.tsx

```diff
@@ -73,14 +90,17 @@ export function App(): JSX.Element {
         },
         config: resp.data["config"],
       });
+      const debug_data = resp.data["debug"];
```

Comment:
- will the resp always contain a "debug" field? If not, we should check that debug_data is not null or undefined before accessing its fields

- variable names should be snakeCase

Reply:
Even though I expect the "debug" field to always be there, this is a good check to have. Added it in. And yes, snakeCase done :) Thanks!